tests/test_meta.py

```diff
@@ -254,6 +256,50 @@ def test_commit_files(data_archive, cli_runner):
         assert r.exit_code == 44, r.stderr
 
 
+def test_commit_files_add_and_delete(data_working_copy, cli_runner):
+    with data_working_copy("point-cloud/auckland") as (repo_dir, wc):
+        # Set the environment variable
+        os.environ["X_KART_ATTACHMENTS"] = "true"
+        # Define file path
+        file_path = pathlib.Path(repo_dir) / "my_attachment.txt"
+
+        # Make sure the file does not exist yet
+        assert not file_path.exists(), r.stderr
```

Comment:
Remove r.stderr - I don't think r is defined at this point

Basically any time you run a command that you expect to succeed, you can do this:
`assert r.exit_code == 0, r.stderr`
and for other assertions, you shouldn't be referring to whatever is in r.stderr since it may or may not be relevant. You don't necessarily need an extra message for other assertions - you can if you like add a bit of extra info - eg the value of file_path - but you'll see that in most places we haven't bothered